source/declarations.tex

```diff
@@ -8505,6 +8505,9 @@
 the rules specifying to which entity or statement the attribute can apply or
 the syntax rules for the attribute's \grammarterm{attribute-argument-clause}, if any.
 \end{note}
+\begin{note}
+The \grammarterm{attribute}s specified in \ref{dcl.attr} have optional semantics: given a well-formed program, removing all instances of a particular such \grammarterm{attribute} results in a program whose observable behavior is a conforming realization of the original program.
```

Comment:
```suggestion
The \grammarterm{attribute}s specified in \ref{dcl.attr} have optional semantics: given a well-formed program, removing all instances of a particular such \grammarterm{attribute} results in a program whose set of possible executions ([intro.abstract]) is a subset of those of the original program.
```

Reply:
` ([intro.abstract])` should be `iref{intro.abstract}`.